src/packages/sdk/src/predefinedGeneratorResolvers.ts

```diff
@@ -1,14 +1,13 @@
+import Debug from '@prisma/debug'
 import chalk from 'chalk'
 import execa from 'execa'
 import fs from 'fs'
 import hasYarn from 'has-yarn'
 import path from 'path'
+import resolvePkg from 'resolve-pkg'
 import { logger } from '.'
 import { getCommandWithExecutor } from './getCommandWithExecutor'
-import { findUpAsync as findUp } from './utils/find'
```

Comment:
I think this was the only place we used this file/function. So we could then remove it?

Export 
https://github.com/prisma/prisma/blob/master/src/packages/client/src/runtime/index.ts#L33
File
https://github.com/prisma/prisma/blob/master/src/packages/client/src/runtime/utils/find.ts

Reply:
We still use this in `client` to ensure that it finds the generated client for contexts where we are not able to resolve it (eg. webpack, nextjs). We don't need it anymore in `sdk`.